src/constants/chains.ts

```diff
@@ -1,39 +1,51 @@
 import { ChainInfo, SupportedChainId } from "../types";
 
 export const CHAIN_INFO: { [chainId in SupportedChainId]: ChainInfo } = {
-  [SupportedChainId.MAINNET]: {
-    label: "Ethereum",
-    appUrl: "https://looksrare.org",
-    explorer: "https://etherscan.io",
-    rpcUrl: `https://eth-mainnet.g.alchemy.com/v2`,
-    apiUrl: "https://graphql.looksrare.org/graphql",
-    osApiUrl: "https://api.opensea.io",
-    cdnUrl: "https://static.looksnice.org",
-    rewardsSubgraphUrl: "https://api.thegraph.com/subgraphs/name/looksrare/looks-distribution",
-    cloudinaryUrl: "https://looksrare.mo.cloudinary.net",
-  },
-  [SupportedChainId.GOERLI]: {
-    label: "Goerli",
-    appUrl: "https://goerli.looksrare.org",
-    explorer: "https://goerli.etherscan.io",
-    rpcUrl: `https://eth-goerli.g.alchemy.com/v2`,
-    apiUrl: "https://graphql-goerli.looksrare.org/graphql",
-    osApiUrl: "https://testnets-api.opensea.io",
-    cdnUrl: "https://static-goerli.looksnice.org",
-    rewardsSubgraphUrl: "https://api.thegraph.com/subgraphs/name/0xjurassicpunk/looks-distribution",
-    cloudinaryUrl: "https://looksrare.mo.cloudinary.net/goerli",
-  },
-  [SupportedChainId.HARDHAT]: {
-    label: "Hardhat",
-    appUrl: "http://localhost:3000",
-    explorer: "https://etherscan.io",
-    rpcUrl: "http://127.0.0.1:8545",
-    apiUrl: "http://localhost:4000/graphql",
-    osApiUrl: "https://testnets-api.opensea.io",
-    cdnUrl: "https://via.placeholder.com",
-    rewardsSubgraphUrl: "https://api.thegraph.com/subgraphs/name/0xjurassicpunk/looks-distribution",
-    cloudinaryUrl: "",
-  },
+  [SupportedChainId.MAINNET]: (() => {
+    const baseApiUrl = "https://graphql.looksrare.org";
```

Comment:
Could just declare 3 consts outside of the object for each base url (`MAINNET_BASE_API_URL`... etc.)

Any reason you opted for changing + calling the function rather than doing that?

Reply:
No strong feelings about that, just personally prefer having them within the scope they're used in.
IIFEs are a usually a controversial topic tho, so open to change that if requested 😆